src/context.c

```diff
@@ -315,6 +316,15 @@ static gboolean r_context_configure_target(GError **error)
 		}
 	}
 
+	/* load system status if central status file is available */
+	if (g_strcmp0(context->config->statusfile_path, "per-slot") != 0) {
+		g_clear_pointer(&context->system_status, r_system_status_free);
+		context->system_status = g_new0(RSystemStatus, 1);
+		if (!r_system_status_load(context->config->statusfile_path, context->system_status, &ierror)) {
```

Comment:
`ierror` needs to be cleared on in the error case, otherwise the next function can't use it.

Reply:
Will fix